src/discord/discord.module.ts

```diff
@@ -2,9 +2,10 @@ import { Module } from '@nestjs/common';
 import { DiscordService } from './discord.service';
 import { ConfigModule } from '@nestjs/config';
 import { CabinModule } from 'src/cabin/cabin.module';
+import { SlashCommandModule } from 'src/slashCommand/slashCommand.module';
 
 @Module({
-  imports: [ConfigModule, CabinModule],
+  imports: [ConfigModule, CabinModule, SlashCommandModule], // TODO SlashCommandModule is not needed every time?!
```

Comment:
Do we need this import here? I think it would be better if slash commands depended on discord service rather than the other way. I think you could move this import to the app module instead.

Reply:
Done!